Sorting capture devices by last-updated-timestamp isn't possible

wsmirnow opened the pull request:
Let's disable the sorting table by this column.

Fixes #1209

stefanosgeo commented:
I don't think this is a good idea. In some cases, sorting the CAs by their 'last updated' timestamp makes more sense—because although the status might show 'online', the CA could actually be offline, and the status hasn't been updated yet.

![Screenshot_20250513_161623](https://github.com/user-attachments/assets/19ef509c-08b9-4168-8092-43960a5c6d9f)

Arnei commented:
I'm inclined to agree with Stefanos. We have other date columns that are sortable, I see no reason to make this one in particular unsortable.

gregorydlogan commented:
@wsmirnow see my comment on #1209: rest call is sorting on `update`, needs to be `updated`.

wsmirnow commented:
> @wsmirnow see my comment on #1209: rest call is sorting on `update`, needs to be `updated`.

Thank you @gregorydlogan, I've updated this PR accordingly. The sorting by update**d** collumn is enabled again and fixed by using same naming as OC backend.

wsmirnow commented:
![image](https://github.com/user-attachments/assets/53071dc4-ac4c-4219-92a6-71959bb23baa)

stefanosgeo commented:
@wsmirnow  I can review this...

stefanosgeo commented:
I proxied the PR with our dev cluster,  but I’m unable to sort the CAs. Did I do something wrong?

[2025-05-15 14-27 (Display, OC Studio).webm](https://github.com/user-attachments/assets/20f1a38e-6775-473d-ab10-fcab82795749)

wsmirnow commented:
> I proxied the PR with our dev cluster, but I’m unable to sort the CAs. Did I do something wrong?
> 
>  2025-05-15.14-27.Display.OC.Studio.webm

Please remove cookies and try again.

stefanosgeo reviewed (approved):
Sorting per update column is now working, and code change looks good
